### PR TITLE
v0.11.0

### LINKED ISSUE #734: Release learnr 0.11.0
Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_learnr.html)
* [x] Check if any deprecation processes should be advanced, as described in [Gradual deprecation](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] Fix functions with missing `@return`:
  * [x] debug_exercise_checker
  * [x] disable_all_tags
  * [x] duplicate_env
  * [x] external_evaluator
  * [x] finalize_question
  * [x] initialize_tutorial
  * [x] knit_print
  * [x] mock_exercise
  * [x] one_time
  * [x] question_methods
  * [x] quiz
  * [x] run_tutorial
  * [x] safe_env
  * [x] safe
  * [x] tutorial_html_dependency
  * [x] tutorial_options
  * [x] tutorial
* [x] Fix missing `@examples`:
  * [x] answer
  * [x] ~debug_exercise_checker~
  * [x] ~event_register_handler~
  * [x] ~external_evaluator~
  * [x] ~filesystem_storage~
  * [x] ~get_tutorial_state~
  * [x] ~initialize_tutorial~
  * [x] ~knit_print~
  * [x] question_methods
  * [x] random_praise
  * [x] safe_env
  * [x] ~tutorial_html_dependency~
  * [x] tutorial_options
  * [x] tutorial
* [x] Remove or hide examples in internal functions:
  * [x] mock_exercise.Rd,
  * [x] py_clear_env.Rd,
  * [x] py_copy_global_env.Rd
* [x] `devtools::build_readme()`
* [x] `urlchecker::url_check()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `revdepcheck::cloud_check()`
* [x] Update `cran-comments.md`
* [x] `git push`
* [x] Draft blog post
* [ ] Slack link to draft blog in #open-source-comms

Submit to CRAN:

* [x] `usethis::use_version('minor')`
* [x] `devtools::submit_cran()`
* [x] Approve email
* [x] Maintainer update?

Wait for CRAN...

* [x] Accepted :tada:
* [ ] `git push`
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version()`
* [ ] `git push`
* [x] Finish blog post
* [ ] Tweet
* [x] Add link to blog post in pkgdown news menu

### PR DESCRIPTION
**Pull request by gadenbuie**
Fixes #734  and #730

### CODE REVIEW

**Comment by gadenbuie** [comment]
Release post edits needed

- [ ] "focus on practice" → practicing
- [ ] Add post description for social cards and metadata

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+
+Read on for an overview of the changes in version `0.11.0`,
+or review the [changelog](../../news.index.html) for a full list of updates.
+
+## learnr speaks more than R
+
+learnr tutorials are a great way to teach others R:
+it's in the package name, after all.
+And thanks to R Markdown's flexibility,
+learnr a great way to teach other programming languages as well,
```
Comment (rossellhayes) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:37]:
```suggestion
learnr is a great way to teach other programming languages as well,
```

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+
+### Support for additional programming languages
+
+In addition to spoken languages,
+learnr is now better at running code in programming languages other than R.
+The biggest improvement is for SQL exercises,
+where learners can execute SQL queries on a database.
+This was previously possible,
+but now tutorial authors can use grading packages like [gradethis]
+to grade the tables return by the student's queries.
```
Comment (rossellhayes) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:101]:
```suggestion
to grade the tables returned by the student's queries.
```

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+You can try this in action using 
+`run_tutorial("sql-exericse", "learnr")`
```
Comment (rossellhayes) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:103]:
```suggestion
`run_tutorial("sql-exercise", "learnr")`
```

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+`run_tutorial("polyglot", "learnr")` 
+([polyglot online version](https://learnr-examples.shinyapps.io/polyglot/))
+to see several programming languages in use in the same tutorial.
+
+```{=html}
+<img src = "images/r-python-exercise.png" alt="One R and one Python interactive exercise component, with the same source code but which evaluates differently in each language. The code is: 
+
+x = 5
+x <- 10
+print(x)" />
```
Comment (rossellhayes) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:119]: Love this example!

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 5]
```diff
@@ -0,0 +1,435 @@
...
+
+## Exercises
+
+Beyond expanded language support,
+interactive exercises and questions in learnr tutorials
+have received a number of updates and improvements.
+
+### Setup chunk chaining
+
+Exercises can now be chained together via changed setup chunks
```
Comment (rossellhayes) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:134]:
```suggestion
Exercises can now be chained together via chained setup chunks
```
Not positive that this is a typo, but "changed" didn't make sense to me here.

**Review by rossellhayes — DISMISSED** [review]
This looks great, I just wanted to mark a few typos I caught in the release blog post!

File: vignettes/articles/releases/learnr_v0-11-0.Rmd [hunk 6]
```diff
@@ -0,0 +1,435 @@
...
+### Support for additional programming languages
+
+In addition to spoken languages,
+learnr is now better at running code in programming languages other than R.
+The biggest improvement is for SQL exercises,
+where learners can execute SQL queries on a database.
+This was previously possible,
+but now tutorial authors can use grading packages like [gradethis]
+to grade the tables return by the student's queries.
+You can try this in action using
```
Comment (gadenbuie) [inline vignettes/articles/releases/learnr_v0-11-0.Rmd:102]:
```suggestion
You can see this in action using 
```

File: vignettes/articles/releases/learnr_v0-11-0.Rmd (hunk 5 above)
Comment (gadenbuie) [reply vignettes/articles/releases/learnr_v0-11-0.Rmd:134]: Totally a typo, thanks!

File: vignettes/articles/releases/learnr_v0-11-0.Rmd (hunk 4 above)
Comment (gadenbuie) [reply vignettes/articles/releases/learnr_v0-11-0.Rmd:122]: Thanks to @nischalshrestha!

**Pull request merged**